### DISCUSSION TITLE
youtube channel videos

### DISCUSSION THREAD

**Discussion by jeffalo** [ScratchAddons/ScratchAddons#1305] [Discussion]
ok so now SA has an official channel. https://www.youtube.com/channel/UCxS_mxVIz4iYGK3DKaupbMA/

currently i was thinking of doing some update changelog videos, but it would also be cool to do other things?

perhaps developer interviews? idk, stuff that can allow us to have better interaction and transperency with users.

any ideas?

**Comment by WorldLanguages** [comment]
I'd love to see 1 minute videos going through the changelog.

**Comment by TheColaber** [comment]
Right now we currently have one video, our extension's trailer.
Ideas We've had in mind were
- a second trailer
- changelog updates (W_L)
- Interviews? (Jeffalo)
- Access Easter Eggs?
- Addon tutorials?

**Comment by anatolefr** [comment]
I saw this channel but I think that the all orange banner could be replaced by a banner containing elements, drawings...? 
If you want me to help ?
